requirements.txt

```diff
@@ -13,7 +13,7 @@ python-coveralls==2.9.3
 tox==3.20.1
 pytest==6.1.2
```

Comment:
`pip` is having trouble resolving the pinned dependency.  (See the details of the failed build.) I'm less familiar not entirely sure of the best course of action to resolve.

Reply:
I'll fix it tonight.

Reply:
Many thanks, Vlad!